+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Apr 29 02:49:45 2023 - INFO  - Start to run test.
+ grep 'Protocol 2' /etc/ssh/sshd_config
+ CHECK_RESULT 1 0 0 'check SSL version fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check SSL version fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check SSL version fail'
+ LOG_ERROR 'check SSL version fail'
+ message='check SSL version fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check SSL version fail'
Sat Apr 29 02:49:46 2023 - ERROR - check SSL version fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_ssh_config_001.sh line 33'
+ message='oe_test_check_ssh_config_001.sh line 33'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_ssh_config_001.sh line 33'
Sat Apr 29 02:49:47 2023 - ERROR - oe_test_check_ssh_config_001.sh line 33
+ '[' 0 -eq 1 ']'
+ return 0
+ grep '^\s*SyslogFacility AUTH' /etc/ssh/sshd_config
SyslogFacility AUTHPRIV
+ CHECK_RESULT 0 0 0 'check SSH SyslogFacility fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check SSH SyslogFacility fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ cat /etc/ssh/sshd_config
+ grep '^\s*LogLevel'
+ grep DEBUG
+ CHECK_RESULT 1 0 1 'check SSH SyslogFacility fail'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='check SSH SyslogFacility fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ grep '^\s*X11Forwarding no' /etc/ssh/sshd_config
+ CHECK_RESULT 1 0 0 'check SSH X11Forwarding set fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check SSH X11Forwarding set fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check SSH X11Forwarding set fail'
+ LOG_ERROR 'check SSH X11Forwarding set fail'
+ message='check SSH X11Forwarding set fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check SSH X11Forwarding set fail'
Sat Apr 29 02:49:48 2023 - ERROR - check SSH X11Forwarding set fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_ssh_config_001.sh line 44'
+ message='oe_test_check_ssh_config_001.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_ssh_config_001.sh line 44'
Sat Apr 29 02:49:48 2023 - ERROR - oe_test_check_ssh_config_001.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ grep '^\s*MaxAuthTries' /etc/ssh/sshd_config
+ CHECK_RESULT 1 0 0 'check SSH MaxAuthTries set fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check SSH MaxAuthTries set fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check SSH MaxAuthTries set fail'
+ LOG_ERROR 'check SSH MaxAuthTries set fail'
+ message='check SSH MaxAuthTries set fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check SSH MaxAuthTries set fail'
Sat Apr 29 02:49:49 2023 - ERROR - check SSH MaxAuthTries set fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_ssh_config_001.sh line 48'
+ message='oe_test_check_ssh_config_001.sh line 48'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_ssh_config_001.sh line 48'
Sat Apr 29 02:49:50 2023 - ERROR - oe_test_check_ssh_config_001.sh line 48
+ '[' 0 -eq 1 ']'
+ return 0
++ cat /etc/ssh/sshd_config
++ grep '^\s*MaxAuthTries'
++ awk '{print $2}'
+ getValue=
+ test -eq 3
oe_test_check_ssh_config_001.sh: line 51: test: -eq: unary operator expected
+ CHECK_RESULT 2 0 0 'check SSH MaxAuthTries set number fail'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='check SSH MaxAuthTries set number fail'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'check SSH MaxAuthTries set number fail'
+ LOG_ERROR 'check SSH MaxAuthTries set number fail'
+ message='check SSH MaxAuthTries set number fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check SSH MaxAuthTries set number fail'
Sat Apr 29 02:49:51 2023 - ERROR - check SSH MaxAuthTries set number fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_ssh_config_001.sh line 52'
+ message='oe_test_check_ssh_config_001.sh line 52'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_ssh_config_001.sh line 52'
Sat Apr 29 02:49:52 2023 - ERROR - oe_test_check_ssh_config_001.sh line 52
+ '[' 0 -eq 1 ']'
+ return 0
+ grep '^\s*IgnoreRhosts yes' /etc/ssh/sshd_config
+ CHECK_RESULT 1 0 0 'check SSH IgnoreRhosts set fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check SSH IgnoreRhosts set fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check SSH IgnoreRhosts set fail'
+ LOG_ERROR 'check SSH IgnoreRhosts set fail'
+ message='check SSH IgnoreRhosts set fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check SSH IgnoreRhosts set fail'
Sat Apr 29 02:49:53 2023 - ERROR - check SSH IgnoreRhosts set fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_ssh_config_001.sh line 56'
+ message='oe_test_check_ssh_config_001.sh line 56'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_ssh_config_001.sh line 56'
Sat Apr 29 02:49:53 2023 - ERROR - oe_test_check_ssh_config_001.sh line 56
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Apr 29 02:49:54 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 5
+ test 5 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Apr 29 02:49:55 2023 - ERROR - failed to execute the case.
+ exit 5
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_check_ssh_config_001.sh post_test'
+ message='oe_test_check_ssh_config_001.sh post_test'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'oe_test_check_ssh_config_001.sh post_test'
Sat Apr 29 02:49:56 2023 - INFO  - oe_test_check_ssh_config_001.sh post_test
